app/src/main/res/xml/preferences_network.xml

```diff
@@ -28,6 +22,12 @@
                 android:key="prefParallelDownloads"
                 android:title="@string/pref_parallel_downloads_title"
                 app:useStockLayout="true"/>
+        <SwitchPreference
+                android:defaultValue="false"
+                android:enabled="true"
+                android:key="prefMobileUpdate"
+                android:summary="@string/pref_mobileUpdate_sum"
+                android:title="@string/pref_mobileUpdate_title"/>
```

Comment:
I would move this above the parallel downloads because I think this setting is toggled more often than the number of parallel downloads